arangod/Cluster/ClusterFeature.cpp

```diff
@@ -271,7 +275,8 @@ void ClusterFeature::prepare() {
   _agencyCallbackRegistry.reset(new AgencyCallbackRegistry(agencyCallbacksPath()));
 
   // Initialize ClusterInfo library:
-  ClusterInfo::createInstance(_agencyCallbackRegistry.get());
+  _clusterInfo = std::make_unique<ClusterInfo>(server(), _agencyCallbackRegistry.get());
+  TRI_ASSERT(_clusterInfo);
```

Comment:
can be removed. `make_unique` will throw if it can't create the object. So the assertion will always be true.
```suggestion
```